sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -3209,6 +3222,19 @@ class Dataset[T] private[sql](
     }
   }
 
+  private[sql] def getRowsToPython(
```

Comment:
In DataFrameSuite, we have multiple test cases for `showString` instead of `getRows `, which is introduced in this PR. 

We also need the unit test cases for `getRowsToPython`.

Reply:
Got it, the follow up pr I'm working on will add more test for `getRows` and `getRowsToPython`.